fgcm/fgcmNumbaUtilities.py

```diff
@@ -1,6 +1,6 @@
 try:
     from numba import jit
-    has_numba = True
+    has_numba = False
 except ImportError:
```

Comment:
Commit message is long.  Could do:
```
Turn off numba usage since it seems to be slower

This is especially evident with new (i.e. version 1.25+) numpy.
```

Reply:
Commit is already pushed to fgcm master and released so ...